lesson1.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<style>
+		#header {
+			padding: 40px;
+			text-align: center;
+			color: white;
+			background: #666666;
+			font-weight: bold;
+			font-size: 50px;
+		}
+		#Footer {
```

Comment:
```suggestion
		#footer {
```
class để chữ thường e ơi